zilliqa/src/message.rs

```diff
@@ -116,10 +116,17 @@ impl NewView {
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct BlockRequest(pub BlockRef);
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct BlocksRequest(pub BlockRef);
```

Comment:
I would rename this to something like `BlockBatchRequest` - just BlocksRequest looks confusingly similar to BlockRequest, and also at first made me think it would take some parameters specifying the range of blocks as opposed to just a predetermined batch.

Also maybe a doc comment specifying that it's a batch of 100

Reply:
I will rename - but I would not specify 100 in a doc comment as it is up to the sending node and so could really be any number.